doc/image-from-registry.md

```diff
@@ -67,3 +68,62 @@ spec:
         storage: 5Gi
 ```
 Full example is available here: [registry-image-pvc](../manifests/example/registry-image-datavolume.yaml)
+
+# Registry security
+
+## Private registry
+
+If your docker registry requires authentication:
+
+Create a `Secret` in the same namespace as the DataVolume to store user credentials.  See [endpoint-secret](../manifests/example/endpoint-secret.yaml)
+
+Add `SecretRef` to `DataVolume` spec.
+
+```yaml
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: DataVolume
+...
+spec:
+  source:
+    registry: 
+      url: "docker://my-private-registry:5000/my-username/my-image"
+      secretRef: my-docker-creds 
+...
+```
+
+## TLS certificate configuration
+
+If your registry TLS certificate is not signed by an untrusted CA:
```

Comment:
```
If your registry TLS certificate is not signed by an untrusted CA
```
There is a double negative here, should it be If your registry TLS certificate is signed by an untrusted CA?